### PR TITLE
Update is paused state after connect to device

### PR DESCRIPTION
**Pull request by kylebonnici**

### CODE REVIEW

**Comment by kylebonnici** [comment]
Fixes this https://nordicsemi.atlassian.net/browse/NCD-141?jql=project%20%3D%20%22NCD%22%20AND%20component%20%3D%20%22ncd-rssi%22

File: src/features/useRssiDevice.ts [hunk 1]
```diff
@@ -32,6 +33,8 @@ export default () => {
                     getState().app.rssi.scanRepeat
                 );
             });
+            dispatch(setIsPaused(false));
```
Comment (KievDevel) [inline src/features/useRssiDevice.ts:36]: Looks good, but what if setting "initial state" after device is selected?
Whenever "initial state" is updated you don't need to think about new properties that should be updated

**Review by KievDevel — APPROVED** [review]
Approved as it will fix current issue anyway.

File: src/features/useRssiDevice.ts (hunk 1 above)
Comment (kylebonnici) [reply src/features/useRssiDevice.ts:36]: But initial state will replace the side panel settings user might have made

Comment (KievDevel) [reply src/features/useRssiDevice.ts:36]: yep, ideally they should be separated. like 
`{
 initialState: {...}
 filters: {...}
`

**Pull request merged**